test/Recruitment.js

```diff
@@ -15,101 +18,120 @@ describe("Recruitment contract", function () {
   });
 
-  it("Account balances equals 15,000 after withdraw of 10,000", async function() {
-    const { rInstance, tInstance, owner, addr1, addr2, DAI } = await loadFixture(deployRecruitmentFixture);
-    await tInstance.connect(addr1).approve(
-        rInstance.address,
-        `1000${"0".repeat(18)}`
-      );
-    await rInstance.connect(addr1).setPercentages(75,50,20);
-    await tInstance.connect(addr1).approve(
-        rInstance.address,
-        `25000${"0".repeat(18)}`
-      );
-    await rInstance.connect(addr1).setFinalDeposit(DAI,`25000${"0".repeat(18)}`,0);
-    await rInstance.connect(owner).withdrawTokens(DAI,`1000${"0".repeat(18)}`);
+  it("Account balances equals 15,000 after withdraw of 10,000", async function () {
+    const { rInstance, tInstance, owner, addr1, addr2, DAI } =
+      await loadFixture(deployRecruitmentFixture);
+    await tInstance
+      .connect(addr1)
+      .approve(rInstance.address, `1000${"0".repeat(18)}`);
+    await rInstance.connect(addr1).setPercentages(75, 50, 20);
+    await tInstance
+      .connect(addr1)
+      .approve(rInstance.address, `25000${"0".repeat(18)}`);
+    await rInstance
+      .connect(addr1)
+      .setFinalDeposit(DAI, `25000${"0".repeat(18)}`, 0);
+    await rInstance.connect(owner).withdrawTokens(DAI, `1000${"0".repeat(18)}`);
     const rBalance = await rInstance.accountBalances(addr1.address, DAI);
     expect(rBalance).to.equal(`15000${"0".repeat(18)}`);
   });
 
-//   it("Final payment with wrong initial deposit index fails", async function() {
-//     const { rInstance, tInstance, owner, addr1, addr2, DAI } = await loadFixture(deployRecruitmentFixture);
-//     await tInstance.connect(addr1).approve(
-//         rInstance.address,
-//         `1000${"0".repeat(18)}`
-//       );
-//     await rInstance.connect(addr1).setPercentages(DAI,50,20);
-//     await tInstance.connect(addr1).approve(
-//         rInstance.address,
-//         `24000${"0".repeat(18)}`
-//       );
-//     expect(await rInstance.connect(addr1).setFinalDeposit(DAI,`24000${"0".repeat(18)}`,1)).to.revertedWith("Initial deposit index does not match!");
-//   });
-});
\ No newline at end of file
+  it("Final payment with wrong initial deposit index fails", async function () {
+    const { rInstance, tInstance, owner, addr1, addr2, DAI } =
+      await loadFixture(deployRecruitmentFixture);
+    await tInstance
+      .connect(addr1)
+      .approve(rInstance.address, `1000${"0".repeat(18)}`);
+    await rInstance.connect(addr1).setPercentages(DAI, 50, 20);
+    await tInstance
+      .connect(addr1)
+      .approve(rInstance.address, `24000${"0".repeat(18)}`);
+    expect(
+      await rInstance
+        .connect(addr1)
+        .setFinalDeposit(DAI, `24000${"0".repeat(18)}`, 1)
+    ).to.revertedWith("Initial deposit index does not match!");
+  });
+
+  // company setting scores for a candidate
+  it("A Company should be able to send scores to a candidate", async function () {
```

Comment:
Cheers for the test. Very thoughtful. If you update the above code please adjust your test :)

Reply:
Done updated the test